opensearchpy/connection/krb_requests.py

```diff
@@ -0,0 +1,64 @@
+"""
+Class for the kerberos authentication.
```

Comment:
Nit: "Implements a connection with Kerberos authentication support." (cause we know it's a class ;))